leave_tracker/models.py

```diff
@@ -6,15 +6,15 @@
 from django.core.mail import send_mail
 from django.conf import settings
 from django.template.loader import render_to_string
-
+from django.core.validators import MaxValueValidator
 
 from datetime import timedelta
 
 
 class LeaveCategory(models.Model):
     "The type of leaves. Eg Casual leave, medical leave"
     type_of_leave = models.CharField(max_length=20)
-    number_of_days = models.IntegerField(max_length=10)
+    number_of_days = models.PositiveIntegerField(validators=[MaxValueValidator(9999999999)])
```

Comment:
@omkarvijay5 @shivakrshn49  Even me too thinking of this. Another possibility is One person can't take leave for 366 days right? So better we should limit the number of days to at least one month i.e., 30 days. Sounds good?